std/typecons.d

```diff
@@ -759,43 +771,131 @@ template Tuple(Specs...)
             return h;
         }
 
-        void toString(DG)(scope DG sink)
+        template toString()
```

Comment:
This needs at least an empty documentation comment (e.g. `///`) for the documentation for the nested symbols to show up.